aws-kendra-faq/aws-kendra-faq.json

```diff
@@ -46,6 +46,15 @@
       "minLength": 1,
       "maxLength": 1000
     },
+    "FileFormat": {
```

Comment:
np; I'd probably name this `FaqFileFormat` so it's clear this is only for FAQ as I think the generated enum in the SDK is called.

Reply:
Good point about the enum name. This is in the FAQ provider so it should be clear it is for FAQ. Also the field in CreateFaqRequest is named FileFormat.